libs/chatbot/src/lib/ui/styles.less

```diff
@@ -0,0 +1 @@
+@import '../../../../designer-ui/src/styles.less';
```

Comment:
we shouldn't be importing the designer-ui less files, we should instead be importing any designer-ui components